docs/topics/db/queries.txt

```diff
@@ -496,33 +496,36 @@ is ``'Beatles Blog'``::
 This spanning can be as deep as you'd like.
 
 It works backwards, too. To refer to a "reverse" relationship, use the
-lowercase name of the model.
+:attr:`~.ForeignKey.related_name` of the relationship, by default the lowercase
+name of the model plus ``_set``, e.g. ``entry_set``. See the :ref:`related
+objects documentation (below) <backwards-related-objects>` for a full
+explanation.
```

Comment:
Reverse managers use `<lowercased model name>_set`, but lookups use `<lowercased model name>`. We can remove the first commit.

Reply:
Yes. In that case this isn't right at all. 

Let me think about it and re-phrase.

Reply:
OK, Take 2 in fe2798090bf8b69723d8ddf67d9f8b31f5bb6254. If we don't like something like that then maybe just a wontfix...